index.d.ts

```diff
@@ -0,0 +1 @@
+export default type Element = HTMLElement;
```

Comment:
I'm pretty sure this is not correct. At least I wanted to have a specifc type imported from `ember-element-helper` to be used for TS interfaces as shown in the readme